components/mpas-ocean/src/shared/mpas_ocn_tendency.F

```diff
@@ -440,7 +578,7 @@ subroutine ocn_tend_vel(domain, tendPool, statePool, forcingPool, &
       endif
 
       ! Add horizontal mixing
-      call ocn_vel_hmix_tend(divergence, relativeVorticity, tendVel,err)
+      call ocn_vel_hmix_tend(divergence, relativeVorticity, viscosity, tendVel,err)
```

Comment:
note that having the 'viscosity' array appear here may be a hold over from the two equation turbulence model code.